csfieldguide/templates/interactives/sorting-algorithms.html

```diff
@@ -0,0 +1,197 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+
+
+
+  <div id="sorting-algorithms-interactive" class="row mt-3">
+    <div class="col-12 text-center">
+      <h1 id="heading">{% trans "Sorting Algorithms" %}</h1>
+    </div>
+    <div id="content" class="container">
+      
+      <div id="sorting-algorithms-interactive-item-unsorted-row-1" class="sorting-algorithms-interactive-container">
```

Comment:
The next ~130 lines seem to have a lot of repeated content, could it be simplified by a loop?

Reply:
I contemplated that idea but I thought that could make the readability of the template really messy? There is a lot of repetition because there are 26 divs, but they are split into 4 rows of 8,8,2,8 so would already take 4 for loops.

Reply:
All good, just an idea.